### ISSUE TITLE
Pretty need an emergancy help

### ISSUE THREAD

**Issue by dsanying** [hexojs/hexo#4172]
## Check List

Please check followings before submitting a new issue.

- [Y] I have already read [Docs page](https://hexo.io/docs/) & [Troubleshooting page](https://hexo.io/docs/troubleshooting)
- [Y] I have already searched existing issues and they are not help to me
- [Y] I examined error or warning messages and it's difficult to solve
- [Y] Using [the latest](https://github.com/hexojs/hexo/releases) version of Hexo (run `hexo version` to check)
- [Y] Node.js is higher than 8.6.0

## Environment & Settings

**Node.js & npm version**

```
node: v12.16.1
npm: v6.13.4

```

**Your site `_config.yml`** (Optional)

```
# Hexo Configuration
## Docs: https://hexo.io/docs/configuration.html
## Source: https://github.com/hexojs/hexo/

# Site
title: 暗影博客
subtitle: <span id="subtitle"> </span>
description: 
keywords:
author: 暗影
language: zh-CN
timezone:

# URL
## If your site is put in a subdirectory, set url as 'http://yoursite.com/child' and root as '/child/'
url: https://dsanying.github.io、
root: /
permalink: posts/:title/
permalink_defaults:
pretty_urls:
  trailing_index: true # Set to false to remove trailing 'index.html' from permalinks
  trailing_html: true # Set to false to remove trailing '.html' from permalinks

# Directory
source_dir: source
public_dir: public
tag_dir: tags
archive_dir: archives
category_dir: categories
code_dir: downloads/code
i18n_dir: :lang
skip_render:

# Writing
new_post_name: :title.md # File name of new posts
default_layout: post
titlecase: false # Transform title into titlecase
external_link:
  enable: false # Open external links in new tab
  field: site # Apply to the whole site
  exclude: ''
filename_case: 0
render_drafts: false
post_asset_folder: false
relative_link: false
future: true
highlight:
  enable: true
  line_number: true
  auto_detect: false
  tab_replace: ''
  wrap: true
  hljs: false

# Home page setting
# path: Root path for your blogs index page. (default = '')
# per_page: Posts displayed per page. (0 = disable pagination)
# order_by: Posts order. (Order by date descending by default)
index_generator:
  path: ''
  per_page: 10
  order_by: -date

# Category & Tag
default_category: uncategorized
category_map:
tag_map:

# Metadata elements
## https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta
meta_generator: true

# Date / Time format
## Hexo uses Moment.js to parse and display date
## You can customize the date format as defined in
## http://momentjs.com/docs/#/displaying/format/
date_format: YYYY-MM-DD
time_format: HH:mm:ss
## Use post's date for updated date unless set in front-matter
use_date_for_updated: false

# Pagination
## Set per_page to 0 to disable pagination
per_page: 10
pagination_dir: page

# Include / Exclude file(s)
## include:/exclude: options only apply to the 'source/' folder
include:
exclude:
ignore:

# Extensions
## Plugins: https://hexo.io/plugins/
## Themes: https://hexo.io/themes/
theme: next

# Deployment
## Docs: https://hexo.io/docs/deployment.html
deploy:
  - type: git
    repo:
      github: git@github.com:dsanying/dsanying.github.io.git
    branch: master
  - type: baidu_url_submitter

# hexo-hide-posts
# hide_posts:
#   # 可以改成其他你喜欢的名字
#   filter: hidden
#   # 指定你想要传递隐藏文章的 generator，比如让所有隐藏文章在存档页面可见
#   # 常见的 generators 有：index, tag, category, archive, sitemap, feed, etc.
#   public_generators: []
#   # 为隐藏的文章添加 noindex meta 标签，阻止搜索引擎收录
#   noindex: true

# hexo-blog-encrypt
encrypt:
  abstract: The content is encrypted!Enter the password to read!
  message: Please enter the password：
  tags:
  - {name: encrypt, password: unlock}
  template: <div id="hexo-blog-encrypt" data-wpm="{{hbeWrongPassMessage}}" data-whm="{{hbeWrongHashMessage}}"><div class="hbe-input-container"><input type="password" id="hbePass" placeholder="{{hbeMessage}}" /><label>{{hbeMessage}}</label><div class="bottom-line"></div></div><script id="hbeData" type="hbeData" data-hmacdigest="{{hbeHmacDigest}}">{{hbeEncryptedData}}</script></div>
  wrong_pass_message: Wrong password!Please enter the password again!
  wrong_hash_message: Sorry,the content can't be hash!

# hexo-tag-aplayer
aplayer:
  meting: true

# 百度推送
baidu_url_submit:
  count: 0                  # 提交最新的多少个链接
  host: dsanying.github.io    # 在百度站长平台中添加的域名
  token: 4kU3vgf8y5pM67aQ     # 秘钥
  path: baidu_urls.txt   # 文本文档的地址， 新链接会保存在此文本文档里

# hexo-git-backup
backup:
  type: git
  theme: next
  message: Back up dsanying's blog
  repository:
    github: git@github.com:dsanying/dsanying.github.io.git,backup
```

**Your theme `_config.yml`** (Optional)

```
Theme: next

# ---------------------------------------------------------------
# Theme Core Configuration Settings
# See: https://theme-next.org/docs/theme-settings/
# ---------------------------------------------------------------

# If false, merge configs from `_data/next.yml` into default configuration (rewrite).
# If true, will fully override default configuration by options from `_data/next.yml` (override). Only for NexT settings.
# And if true, all config from default NexT `_config.yml` must be copied into `next.yml`. Use if you know what you are doing.
# Useful if you want to comment some options from NexT `_config.yml` by `next.yml` without editing default config.
override: false

# Console reminder if new version released.
reminder: true

# Allow to cache content generation. Introduced in NexT v6.0.0.
cache:
  enable: true

# Remove unnecessary files after hexo generate.
minify: false

# Define custom file paths.
# Create your custom files in site directory `source/_data` and uncomment needed files below.
custom_file_path:
  #head: source/_data/head.swig
  #header: source/_data/header.swig
  #sidebar: source/_data/sidebar.swig
  #postMeta: source/_data/post-meta.swig
  #postBodyEnd: source/_data/post-body-end.swig
  #footer: source/_data/footer.swig
  #bodyEnd: source/_data/body-end.swig
  #variable: source/_data/variables.styl
  #mixin: source/_data/mixins.styl
  #style: source/_data/styles.styl


# ---------------------------------------------------------------
# Site Information Settings
# See: https://theme-next.org/docs/getting-started/
# ---------------------------------------------------------------

favicon:
  small: /images/favicon-16x16-next.png
  medium: /images/favicon-32x32-next.png
  apple_touch_icon: /images/apple-touch-icon-next.png
  safari_pinned_tab: /images/logo.svg
  #android_manifest: /images/manifest.json
  #ms_browserconfig: /images/browserconfig.xml

footer:
  # Specify the date when the site was setup. If not defined, current year will be used.
  since: 2019

  # Icon between year and copyright info.
  icon:
    # Icon name in Font Awesome. See: https://fontawesome.com/v4.7.0/icons/
    # `heart` is recommended with animation in red (#ff0000).
    name: user
    # If you want to animate the icon, set it to true.
    animated: false
    # Change the color of icon, using Hex Code.
    color: "#808080"

  # If not defined, `author` from Hexo `_config.yml` will be used.
  copyright: <a href="https://dsanying.github.io/">暗影</a>

  powered:
    # Hexo link (Powered by Hexo).
    enable: false
    # Version info of Hexo after Hexo link (vX.X.X).
    version: false

  theme:
    # Theme & scheme info link (Theme - NexT.scheme).
    enable: false
    # Version info of NexT after scheme info (vX.X.X).
    version: false

  # Beian ICP and gongan information for Chinese users. See: http://www.beian.miit.gov.cn, http://www.beian.gov.cn
  beian:
    enable: false
    icp:
    # The digit in the num of gongan beian.
    gongan_id:
    # The full num of gongan beian.
    gongan_num:
    # The icon for gongan beian. See: http://www.beian.gov.cn/portal/download
    gongan_icon_url:

# Creative Commons 4.0 International License.
# See: https://creativecommons.org/share-your-work/licensing-types-examples
# Available values of license: by | by-nc | by-nc-nd | by-nc-sa | by-nd | by-sa | zero
# You can set a language value if you prefer a translated version of CC license, e.g. deed.zh
# CC licenses are available in 39 languages, you can find the specific and correct abbreviation you need on https://creativecommons.org
creative_commons:
  license: by-nc-sa
  sidebar: true
  post: true
  language:


# ---------------------------------------------------------------
# Scheme Settings
# ---------------------------------------------------------------

# Schemes
#scheme: Muse
#scheme: Mist
#scheme: Pisces
scheme: Gemini

# Dark Mode
darkmode: true


# ---------------------------------------------------------------
# Menu Settings
# ---------------------------------------------------------------

# Usage: `Key: /link/ || icon`
# Key is the name of menu item. If the translation for this item is available, the translated text will be loaded, otherwise the Key name will be used. Key is case-senstive.
# Value before `||` delimiter is the target link, value after `||` delimiter is the name of Font Awesome icon.
# When running the site in a subdirectory (e.g. yoursite.com/blog), remove the leading slash from link value (/archives -> archives).
# External url should start with http:// or https://
menu:
  home: / || home
  archives: /archives/ || archive
  categories: /categories/ || th
  tags: /tags/ || tags
  links: /links/ || users
  tools: /tools/ || briefcase
  #about: /about/ || user
  #schedule: /schedule/ || calendar
  #sitemap: /sitemap.xml || sitemap
  #commonweal: /404/ || heartbeat

# Enable / Disable menu icons / item badges.
menu_settings:
  icons: true
  badges: false


# ---------------------------------------------------------------
# Sidebar Settings
# See: https://theme-next.org/docs/theme-settings/sidebar
# ---------------------------------------------------------------

sidebar:
  # Sidebar Position.
  #position: left
  position: right

  # Manual define the sidebar width. If commented, will be default for:
  # Muse | Mist: 320
  # Pisces | Gemini: 240
  #width: 300

  # Sidebar Display (only for Muse | Mist), available values:
  #  - post    expand on posts automatically. Default.
  #  - always  expand for all pages automatically.
  #  - hide    expand only when click on the sidebar toggle icon.
  #  - remove  totally remove sidebar including sidebar toggle.
  display: post

  # Sidebar padding in pixels.
  padding: 18
  # Sidebar offset from top menubar in pixels (only for Pisces | Gemini).
  offset: 12
  # Enable sidebar on narrow view (only for Muse | Mist).
  onmobile: false

# Sidebar Avatar
avatar:
  # Replace the default image and set the url here.
  url: https://cdn.jsdelivr.net/gh/dsanying/ImgHosting/avatar.jpg #/images/avatar.gif
  # If true, the avatar will be dispalyed in circle.
  rounded: true
  # If true, the avatar will be rotated with the cursor.
  rotated: true

# Posts / Categories / Tags in sidebar.
site_state: true

# Social Links
# Usage: `Key: permalink || icon`
# Key is the link label showing to end users.
# Value before `||` delimiter is the target permalink, value after `||` delimiter is the name of Font Awesome icon.
social:
  GitHub: https://github.com/dsanying || github
  E-Mail: http://mail.qq.com/cgi-bin/qm_share?t=qm_mailme&email=KhsbEx8eEh8SGWpbWwRJRUc || envelope
  #Weibo: https://weibo.com/yourname || weibo
  #Google: https://plus.google.com/yourname || google
  #Twitter: https://twitter.com/yourname || twitter
  #FB Page: https://www.facebook.com/yourname || facebook
  #StackOverflow: https://stackoverflow.com/yourname || stack-overflow
  #YouTube: https://youtube.com/yourname || youtube
  #Instagram: https://instagram.com/yourname || instagram
  #Skype: skype:yourname?call|chat || skype

social_icons:
  enable: true
  icons_only: true
  transition: false

# Blog rolls
links_settings:
  icon: link
  title: Links
  # Available values: block | inline
  layout: block

links:
  #Title: http://yoursite.com
  分享站: https://dsanying.github.io/cloud
  文件收集: https://inbox.weiyun.com/5TwJzIf
  收集箱文件: https://share.weiyun.com/5EKcjwn


# Table of Contents in the Sidebar
# Front-matter variable (unsupport wrap expand_all).
toc:
  enable: true
  # Automatically add list number to toc.
  number: false
  # If true, all words will placed on next lines if header width longer then sidebar width.
  wrap: false
  # If true, all level of TOC in a post will be displayed, rather than the activated part of it.
  expand_all: false
  # Maximum heading depth of generated toc.
  max_depth: 6

# A button to open designated chat widget in sidebar.
# Firstly, you need enable the chat service you want to activate its sidebar button.
chat:
  enable: false
  #service: chatra
  #service: tidio
  icon: comment # Icon name in Font Awesome, set false to disable icon.
  text: Chat # Button text, change it as you wish.


# ---------------------------------------------------------------
# Post Settings
# See: https://theme-next.org/docs/theme-settings/posts
# ---------------------------------------------------------------

# Automatically excerpt description in homepage as preamble text.
excerpt_description: true

# Read more button
# If true, the read more button will be displayed in excerpt section.
read_more_btn: true

# Post meta display settings
post_meta:
  item_text: true
  created_at: false
  updated_at:
    enable: true
    another_day: true
  categories: true

# Post wordcount display settings
# Dependencies: https://github.com/theme-next/hexo-symbols-count-time
symbols_count_time:
  separated_meta: true
  item_text_post: true
  item_text_total: false
  awl: 4
  wpm: 275

# Use icon instead of the symbol # to indicate the tag at the bottom of the post
tag_icon: true

# Reward (Donate)
# Front-matter variable (unsupport animation).
reward_settings:
  # If true, reward will be displayed in every article by default.
  enable: true
  animation: false
  #comment: Donate comment here.

reward:
  wechatpay: /images/wechatpay.png
  #alipay: /images/alipay.png
  #paypal: /images/paypal.png
  #bitcoin: /images/bitcoin.png

# Subscribe through Telegram Channel, Twitter, etc.
# Usage: `Key: permalink || icon` (Font Awesome)
follow_me:
  #Twitter: https://twitter.com/username || twitter
  #Telegram: https://t.me/channel_name || telegram
  #WeChat: /images/wechat_channel.jpg || wechat
  #RSS: /atom.xml || rss

# Related popular posts
# Dependencies: https://github.com/tea3/hexo-related-popular-posts
related_posts:
  enable: true
  title: # Custom header, leave empty to use the default one
  display_in_home: false
  params:
    maxCount: 5
    #PPMixingRate: 0.0
    #isDate: false
    #isImage: false
    #isExcerpt: false

# Post edit
# Dependencies: https://github.com/hexojs/hexo-deployer-git
post_edit:
  enable: false
  url: https://github.com/user-name/repo-name/tree/branch-name/subdirectory-name # Link for view source
  #url: https://github.com/user-name/repo-name/edit/branch-name/subdirectory-name # Link for fork & edit

# Show previous post and next post in post footer if exists
# Available values: left | right | false
post_navigation: left


# ---------------------------------------------------------------
# Custom Page Settings
# See: https://theme-next.org/docs/theme-settings/custom-pages
# ---------------------------------------------------------------

# TagCloud settings for tags page.
tagcloud:
  # All values below are same as default, change them by yourself.
  min: 12 # Minimun font size in px
  max: 30 # Maxium font size in px
  start: "#ccc" # Start color (hex, rgba, hsla or color keywords)
  end: "#111" # End color (hex, rgba, hsla or color keywords)
  amount: 200 # Amount of tags, change it if you have more than 200 tags

# Google Calendar
# Share your recent schedule to others via calendar page.
calendar:
  calendar_id: <required> # Your Google account E-Mail
  api_key: <required>
  orderBy: startTime
  offsetMax: 24 # Time Range
  offsetMin: 4 # Time Range
  showDeleted: false
  singleEvents: true
  maxResults: 250


# ---------------------------------------------------------------
# Misc Theme Settings
# ---------------------------------------------------------------

# Set the text alignment in posts / pages.
text_align:
  # Available values: start | end | left | right | center | justify | justify-all | match-parent
  desktop: justify
  mobile: justify

# Reduce padding / margin indents on devices with narrow width.
mobile_layout_economy: false

# Android Chrome header panel color ($brand-bg / $headband-bg => $black-deep).
android_chrome_color: "#222"

# Hide sticky headers and color the menu bar on Safari (iOS / macOS).
safari_rainbow: false

# Custom Logo (Do not support scheme Mist)
custom_logo: #/uploads/custom-logo.jpg

codeblock:
  # Code Highlight theme
  # Available values: normal | night | night eighties | night blue | night bright | solarized | solarized dark | galactic
  # See: https://github.com/chriskempson/tomorrow-theme
  highlight_theme: normal
  # Add copy button on codeblock
  copy_button:
    enable: true
    # Show text copy result.
    show_result: true
    # Available values: default | flat | mac
    style: flat

back2top:
  enable: true
  # Back to top in sidebar.
  sidebar: true
  # Scroll percent label in b2t button.
  scrollpercent: true

# Reading progress bar
reading_progress:
  enable: true
  # Available values: top | bottom
  position: top
  color: black
  height: 3px

# Bookmark Support
bookmark:
  enable: false
  # Customize the color of the bookmark.
  color: "#222"
  # If auto, save the reading progress when closing the page or clicking the bookmark-icon.
  # If manual, only save it by clicking the bookmark-icon.
  save: auto

# `Follow me on GitHub` banner in the top-right corner.
github_banner:
  enable: false
  permalink: https://github.com/yourname
  title: Follow me on GitHub


# ---------------------------------------------------------------
# Font Settings
# See: https://theme-next.org/docs/theme-settings/#Fonts-Customization
# ---------------------------------------------------------------
# Find fonts on Google Fonts (https://www.google.com/fonts)
# All fonts set here will have the following styles:
#   light | light italic | normal | normal italic | bold | bold italic
# Be aware that setting too much fonts will cause site running slowly
# ---------------------------------------------------------------
# To avoid space between header and sidebar in scheme Pisces / Gemini, Web Safe fonts are recommended for `global` (and `title`):
# Arial | Tahoma | Helvetica | Times New Roman | Courier New | Verdana | Georgia | Palatino | Garamond | Comic Sans MS | Trebuchet MS
# ---------------------------------------------------------------

font:
  enable: false

  # Uri of fonts host, e.g. https://fonts.googleapis.com (Default).
  host:

  # Font options:
  # `external: true` will load this font family from `host` above.
  # `family: Times New Roman`. Without any quotes.
  # `size: x.x`. Use `em` as unit. Default: 1 (16px)

  # Global font settings used for all elements inside <body>.
  global:
    external: true
    family: Lato
    size:

  # Font settings for site title (.site-title).
  title:
    external: true
    family:
    size:

  # Font settings for headlines (<h1> to <h6>).
  headings:
    external: true
    family:
    size:

  # Font settings for posts (.post-body).
  posts:
    external: true
    family:

  # Font settings for <code> and code blocks.
  codes:
    external: true
    family:


# ---------------------------------------------------------------
# SEO Settings
# ---------------------------------------------------------------

# Disable Baidu transformation on mobile devices.
disable_baidu_transformation: false

# Set a canonical link tag for your Hexo site.
# See: https://support.google.com/webmasters/answer/139066
# Remember to set up your URL in Hexo `_config.yml` (e.g. url: http://yoursite.com)
canonical: true

# Change headers hierarchy on site-subtitle (will be main site description) and on all post / page titles for better SEO-optimization.
seo: false

# If true, site-subtitle will be added to index page.
# Remember to set up your site-subtitle in Hexo `_config.yml` (e.g. subtitle: Subtitle)
index_with_subtitle: false

# Automatically add external URL with Base64 encrypt & decrypt.
exturl: false

# Google Webmaster tools verification.
# See: https://www.google.com/webmasters
google_site_verification:

# Bing Webmaster tools verification.
# See: https://www.bing.com/webmaster
bing_site_verification:

# Yandex Webmaster tools verification.
# See: https://webmaster.yandex.ru
yandex_site_verification:

# Baidu Webmaster tools verification.
# See: https://ziyuan.baidu.com/site
baidu_site_verification:

# Enable baidu push so that the blog will push the url to baidu automatically which is very helpful for SEO.
baidu_push: true


# ---------------------------------------------------------------
# Third Party Plugins & Services Settings
# See: https://theme-next.org/docs/third-party-services/
# You may need to install dependencies or set CDN URLs in `vendors`
# There are two different CDN providers by default:
#   - jsDelivr (cdn.jsdelivr.net), works everywhere even in China
#   - CDNJS (cdnjs.cloudflare.com), provided by cloudflare
# ---------------------------------------------------------------

# Math Formulas Render Support
math:
  # Default (true) will load mathjax / katex script on demand.
  # That is it only render those page which has `mathjax: true` in Front-matter.
  # If you set it to false, it will load mathjax / katex srcipt EVERY PAGE.
  per_page: false

  # hexo-renderer-pandoc (or hexo-renderer-kramed) required for full MathJax support.
  mathjax:
    enable: true
    # See: https://mhchem.github.io/MathJax-mhchem/
    mhchem: false

  # hexo-renderer-markdown-it-plus (or hexo-renderer-markdown-it with markdown-it-katex plugin) required for full Katex support.
  katex:
    enable: false
    # See: https://github.com/KaTeX/KaTeX/tree/master/contrib/copy-tex
    copy_tex: false

# Easily enable fast Ajax navigation on your website.
# Dependencies: https://github.com/theme-next/theme-next-pjax
pjax: false

# FancyBox is a tool that offers a nice and elegant way to add zooming functionality for images.
# For more information: https://fancyapps.com/fancybox
fancybox: false

# A JavaScript library for zooming images like Medium.
# Do not enable both `fancybox` and `mediumzoom`.
# For more information: https://github.com/francoischalifour/medium-zoom
mediumzoom: false

# Vanilla JavaScript plugin for lazyloading images.
# For more information: https://github.com/ApoorvSaxena/lozad.js
lazyload: true

# Pangu Support
# For more information: https://github.com/vinta/pangu.js
pangu: false

# Quicklink Support
# For more information: https://github.com/GoogleChromeLabs/quicklink
# Front-matter (unsupport home archive).
quicklink:
  enable: false

  # Home page and archive page can be controlled through home and archive options below.
  # This configuration item is independent of `enable`.
  home: false
  archive: false

  # Default (true) will initialize quicklink after the load event fires.
  delay: true
  # Custom a time in milliseconds by which the browser must execute prefetching.
  timeout: 3000
  # Default (true) will enable fetch() or falls back to XHR.
  priority: true

  # For more flexibility you can add some patterns (RegExp, Function, or Array) to ignores.
  # See: https://github.com/GoogleChromeLabs/quicklink#custom-ignore-patterns
  ignores:


# ---------------------------------------------------------------
# Comments Settings
# See: https://theme-next.org/docs/third-party-services/comments
# ---------------------------------------------------------------

# Multiple Comment System Support
comments:
  # Available values: tabs | buttons
  style: tabs
  # Choose a comment system to be displayed by default.
  # Available values: changyan | disqus | disqusjs | gitalk | livere | valine
  active:
  # Setting `true` means remembering the comment system selected by the visitor.
  storage: true
  # Lazyload all comment systems.
  lazyload: false
  # Modify texts or order for any navs, here are some examples.
  nav:
    #disqus:
    #  text: Load Disqus
    #  order: -1
    #gitalk:
    #  order: -2

# Disqus
disqus:
  enable: false
  shortname:
  count: true
  #post_meta_order: 0

# DisqusJS
# Alternative Disqus - Render comment component using Disqus API.
# Demo: https://suka.js.org/DisqusJS/
# For more information: https://github.com/SukkaW/DisqusJS
disqusjs:
  enable: false
  # API Endpoint of Disqus API (https://disqus.com/api/).
  # Leave api empty if you are able to connect to Disqus API. Otherwise you need a reverse proxy for it.
  # For example:
  # api: https://disqus.skk.moe/disqus/
  api:
  apikey: # Register new application from https://disqus.com/api/applications/
  shortname: # See: https://disqus.com/admin/settings/general/

# Changyan
changyan:
  enable: false
  appid:
  appkey:
  #post_meta_order: 0

# Valine
# For more information: https://valine.js.org, https://github.com/xCss/Valine
valine:
  enable: true
  appid: QixffAKVaEcoeL9xlvQHE6Px-MdYXbMMI # Your leancloud application appid
  appkey: 0IhgvfF6jTXUK3DBIfiiKK8U # Your leancloud application appkey
  notify: false # Mail notifier
  verify: true # Verification code
  placeholder: 膜拜大佬ヾ(ｏ･ω･)ﾉ # Comment box placeholder
  avatar: mm # Gravatar style
  guest_info: nick,mail,link # Custom comment header
  pageSize: 10 # Pagination size
  language: zh-cn # Language, available values: en, zh-cn
  visitor: false # Article reading statistic
  comment_count: true # If false, comment count will only be displayed in post page, not in home page
  recordIP: true # Whether to record the commenter IP
  serverURLs: # When the custom domain name is enabled, fill it in here (it will be detected automatically by default, no need to fill in)
  #post_meta_order: 0

# LiveRe comments system
# You can get your uid from https://livere.com/insight/myCode (General web site)
livere_uid: # <your_uid>

# Gitalk
# For more information: https://gitalk.github.io, https://github.com/gitalk/gitalk
gitalk:
  enable: false
  github_id: # GitHub repo owner
  repo: # Repository name to store issues
  client_id: # GitHub Application Client ID
  client_secret: # GitHub Application Client Secret
  admin_user: # GitHub repo owner and collaborators, only these guys can initialize gitHub issues
  distraction_free_mode: true # Facebook-like distraction free mode
  # Gitalk's display language depends on user's browser or system environment
  # If you want everyone visiting your site to see a uniform language, you can set a force language value
  # Available values: en | es-ES | fr | ru | zh-CN | zh-TW
  language:


# ---------------------------------------------------------------
# Post Widgets & Content Sharing Services
# See: https://theme-next.org/docs/third-party-services/post-widgets
# ---------------------------------------------------------------

# Star rating support to each article.
# To get your ID visit https://widgetpack.com
rating:
  enable: false
  id:     # <app_id>
  color:  fc6423

# AddThis Share. See: https://www.addthis.com
# Go to https://www.addthis.com/dashboard to customize your tools.
add_this_id:


# ---------------------------------------------------------------
# Statistics and Analytics
# See: https://theme-next.org/docs/third-party-services/statistics-and-analytics
# ---------------------------------------------------------------

# Google Analytics
google_analytics:
  tracking_id: # <app_id>
  # By default, NexT will load an external gtag.js script on your site.
  # If you only need the pageview feature, set the following option to true to get a better performance.
  only_pageview: false

# Baidu Analytics
baidu_analytics: de7ddc19c9165037ffb199862a2641b1 # <app_id>

# Growingio Analytics
growingio_analytics: # <project_id>

# CNZZ count
cnzz_siteid:

# Show number of visitors of each article.
# You can visit https://leancloud.cn to get AppID and AppKey.
# AppID and AppKey are recommended to be the same as valine's for counter compatibility.
# Do not enable both `valine.visitor` and `leancloud_visitors`.
leancloud_visitors:
  enable: false
  app_id: # <app_id>
  app_key: # <app_key>
  # Dependencies: https://github.com/theme-next/hexo-leancloud-counter-security
  # If you don't care about security in leancloud counter and just want to use it directly
  # (without hexo-leancloud-counter-security plugin), set `security` to `false`.
  security: true
  betterPerformance: false

# Another tool to show number of visitors to each article.
# Visit https://console.firebase.google.com/u/0/ to get apiKey and projectId.
# Visit https://firebase.google.com/docs/firestore/ to get more information about firestore.
firestore:
  enable: false
  collection: articles # Required, a string collection name to access firestore database
  apiKey: # Required
  projectId: # Required

# Show Views / Visitors of the website / page with busuanzi.
# Get more information on http://ibruce.info/2015/04/04/busuanzi
busuanzi_count:
  enable: true
  total_visitors: true
  total_visitors_icon: user
  total_views: true
  total_views_icon: eye
  post_views: false
  post_views_icon: eye


# ---------------------------------------------------------------
# Search Services
# See: https://theme-next.org/docs/third-party-services/search-services
# ---------------------------------------------------------------

# Algolia Search
# For more information: https://www.algolia.com
algolia_search:
  enable: false
  hits:
    per_page: 10
  labels:
    input_placeholder: Search for Posts
    hits_empty: "We didn't find any results for the search: ${query}"
    hits_stats: "${hits} results found in ${time} ms"

# Local Search
# Dependencies: https://github.com/theme-next/hexo-generator-searchdb
local_search:
  enable: true
  # If auto, trigger search by changing input.
  # If manual, trigger search by pressing enter key or search button.
  trigger: auto
  # Show top n results per article, show all results by setting to -1
  top_n_per_article: 1
  # Unescape html strings to the readable one.
  unescape: false
  # Preload the search data when the page loads.
  preload: false

# Swiftype Search API Key
swiftype_key:


# ---------------------------------------------------------------
# Chat Services
# See: https://theme-next.org/docs/third-party-services/chat-services
# ---------------------------------------------------------------

# Chatra Support
# See: https://chatra.io
# Dashboard: https://app.chatra.io/settings/general
chatra:
  enable: false
  async: true
  id: # Visit Dashboard to get your ChatraID
  #embed: # Unfinished experimental feature for developers. See: https://chatra.io/help/api/#injectto

# Tidio Support
# See: https://www.tidiochat.com
# Dashboard: https://www.tidiochat.com/panel/dashboard
tidio:
  enable: false
  key: # Public Key, get it from dashboard. See: https://www.tidiochat.com/panel/settings/developer


# ---------------------------------------------------------------
# Tags Settings
# See: https://theme-next.org/docs/tag-plugins/
# ---------------------------------------------------------------

# Note tag (bs-callout)
note:
  # Note tag style values:
  #  - simple    bs-callout old alert style. Default.
  #  - modern    bs-callout new (v2-v3) alert style.
  #  - flat      flat callout style with background, like on Mozilla or StackOverflow.
  #  - disabled  disable all CSS styles import of note tag.
  style: simple
  icons: false
  # Offset lighter of background in % for modern and flat styles (modern: -12 | 12; flat: -18 | 6).
  # Offset also applied to label tag variables. This option can work with disabled note tag.
  light_bg_offset: 0

# Tabs tag
tabs:
  transition:
    tabs: false
    labels: true

# PDF tag, requires two plugins: pdfObject and pdf.js
# pdfObject will try to load pdf files natively, if failed, pdf.js will be used.
# The following `cdn` setting is only for pdfObject, because cdn for pdf.js might be blocked by CORS policy.
# So, you must install the dependency of pdf.js if you want to use pdf tag and make it available to all browsers.
# See: https://github.com/theme-next/theme-next-pdf
pdf:
  enable: true
  # Default height
  height: 500px

# Mermaid tag
mermaid:
  enable: true
  # Available themes: default | dark | forest | neutral
  theme: default


# ---------------------------------------------------------------
# Animation Settings
# ---------------------------------------------------------------

# Use velocity to animate everything.
# For more information: http://velocityjs.org
motion:
  enable: true
  async: false
  transition:
    # Transition variants:
    # fadeIn | flipXIn | flipYIn | flipBounceXIn | flipBounceYIn
    # swoopIn | whirlIn | shrinkIn | expandIn
    # bounceIn | bounceUpIn | bounceDownIn | bounceLeftIn | bounceRightIn
    # slideUpIn | slideDownIn | slideLeftIn | slideRightIn
    # slideUpBigIn | slideDownBigIn | slideLeftBigIn | slideRightBigIn
    # perspectiveUpIn | perspectiveDownIn | perspectiveLeftIn | perspectiveRightIn
    post_block: fadeIn
    post_header: slideDownIn
    post_body: slideDownIn
    coll_header: slideLeftIn
    # Only for Pisces | Gemini.
    sidebar: slideUpIn

# Progress bar in the top during page loading.
# Dependencies: https://github.com/theme-next/theme-next-pace
# For more information: https://github.com/HubSpot/pace
pace:
  enable: false
  # Themes list:
  # big-counter | bounce | barber-shop | center-atom | center-circle | center-radar | center-simple
  # corner-indicator | fill-left | flat-top | flash | loading-bar | mac-osx | material | minimal
  theme: minimal

# JavaScript 3D library.
# Dependencies: https://github.com/theme-next/theme-next-three
three:
  enable: false
  three_waves: false
  canvas_lines: false
  canvas_sphere: false

# Canvas-ribbon
# Dependencies: https://github.com/theme-next/theme-next-canvas-ribbon
# For more information: https://github.com/zproo/canvas-ribbon
canvas_ribbon:
  enable: false
  size: 300 # The width of the ribbon
  alpha: 0.6 # The transparency of the ribbon
  zIndex: -1 # The display level of the ribbon


#! ---------------------------------------------------------------
#! DO NOT EDIT THE FOLLOWING SETTINGS
#! UNLESS YOU KNOW WHAT YOU ARE DOING
#! See: https://theme-next.org/docs/advanced-settings
#! ---------------------------------------------------------------

# Script Vendors. Set a CDN address for the vendor you want to customize.
# Be aware that you would better use the same version as internal ones to avoid potential problems.
# Remember to use the https protocol of CDN files when you enable https on your site.
vendors:
  # Internal path prefix.
  _internal: lib

  # Internal version: 3.1.0
  # anime: //cdn.jsdelivr.net/npm/animejs@3.1.0/lib/anime.min.js
  anime:

  # Internal version: 4.7.0
  # fontawesome: //cdn.jsdelivr.net/npm/font-awesome@4/css/font-awesome.min.css
  # fontawesome: //cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css
  fontawesome:

  # MathJax
  # mathjax: //cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js
  mathjax:

  # KaTeX
  # katex: //cdn.jsdelivr.net/npm/katex@0/dist/katex.min.css
  # katex: //cdnjs.cloudflare.com/ajax/libs/KaTeX/0.11.1/katex.min.css
  # copy_tex_js: //cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.js
  # copy_tex_css: //cdn.jsdelivr.net/npm/katex@0/dist/contrib/copy-tex.min.css
  katex:
  copy_tex_js:
  copy_tex_css:

  # Internal version: 0.2.8
  # pjax: //cdn.jsdelivr.net/gh/theme-next/theme-next-pjax@0/pjax.min.js
  pjax:

  # FancyBox
  # jquery: //cdn.jsdelivr.net/npm/jquery@3/dist/jquery.min.js
  # fancybox: //cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.js
  # fancybox_css: //cdn.jsdelivr.net/gh/fancyapps/fancybox@3/dist/jquery.fancybox.min.css
  jquery:
  fancybox:
  fancybox_css:

  # Medium-zoom
  # mediumzoom: //cdn.jsdelivr.net/npm/medium-zoom@1/dist/medium-zoom.min.js
  mediumzoom:

  # Lazyload
  # lazyload: //cdn.jsdelivr.net/npm/lozad@1/dist/lozad.min.js
  # lazyload: //cdnjs.cloudflare.com/ajax/libs/lozad.js/1.9.0/lozad.min.js
  lazyload:

  # Pangu
  # pangu: //cdn.jsdelivr.net/npm/pangu@4/dist/browser/pangu.min.js
  # pangu: //cdnjs.cloudflare.com/ajax/libs/pangu/4.0.7/pangu.min.js
  pangu:

  # Quicklink
  # quicklink: //cdn.jsdelivr.net/npm/quicklink@1/dist/quicklink.umd.js
  quicklink:

  # DisqusJS
  # disqusjs_js: //cdn.jsdelivr.net/npm/disqusjs@1/dist/disqus.js
  # disqusjs_css: //cdn.jsdelivr.net/npm/disqusjs@1/dist/disqusjs.css
  disqusjs_js:
  disqusjs_css:

  # Valine
  # valine: //cdn.jsdelivr.net/npm/valine@1/dist/Valine.min.js
  # valine: //cdnjs.cloudflare.com/ajax/libs/valine/1.3.10/Valine.min.js
  valine:

  # Gitalk
  # gitalk_js: //cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.min.js
  # gitalk_css: //cdn.jsdelivr.net/npm/gitalk@1/dist/gitalk.css
  gitalk_js:
  gitalk_css:

  # Algolia Search
  # algolia_search: //cdn.jsdelivr.net/npm/algoliasearch@4/dist/algoliasearch-lite.umd.js
  # instant_search: //cdn.jsdelivr.net/npm/instantsearch.js@4/dist/instantsearch.production.min.js
  algolia_search:
  instant_search:

  # PDF
  # pdfobject: //cdn.jsdelivr.net/npm/pdfobject@2/pdfobject.min.js
  # pdfobject: //cdnjs.cloudflare.com/ajax/libs/pdfobject/2.1.1/pdfobject.min.js
  pdfobject:

  # Mermaid
  # mermaid: //cdn.jsdelivr.net/npm/mermaid@8/dist/mermaid.min.js
  # mermaid: //cdnjs.cloudflare.com/ajax/libs/mermaid/8.3.0/mermaid.min.js
  mermaid:

  # Internal version: 1.2.1
  # velocity: //cdn.jsdelivr.net/npm/velocity-animate@1/velocity.min.js
  # velocity: //cdnjs.cloudflare.com/ajax/libs/velocity/1.2.1/velocity.min.js
  # velocity_ui: //cdn.jsdelivr.net/npm/velocity-animate@1/velocity.ui.min.js
  # velocity_ui: //cdnjs.cloudflare.com/ajax/libs/velocity/1.2.1/velocity.ui.min.js
  velocity:
  velocity_ui:

  # Internal version: 1.0.2
  # pace: //cdn.jsdelivr.net/npm/pace-js@1/pace.min.js
  # pace: //cdnjs.cloudflare.com/ajax/libs/pace/1.0.2/pace.min.js
  # pace_css: //cdn.jsdelivr.net/npm/pace-js@1/themes/blue/pace-theme-minimal.css
  # pace_css: //cdnjs.cloudflare.com/ajax/libs/pace/1.0.2/themes/blue/pace-theme-minimal.min.css
  pace:
  pace_css:

  # Internal version: 1.0.0
  # three: //cdn.jsdelivr.net/gh/theme-next/theme-next-three@1/three.min.js
  # three_waves: //cdn.jsdelivr.net/gh/theme-next/theme-next-three@1/three-waves.min.js
  # canvas_lines: //cdn.jsdelivr.net/gh/theme-next/theme-next-three@1/canvas_lines.min.js
  # canvas_sphere: //cdn.jsdelivr.net/gh/theme-next/theme-next-three@1/canvas_sphere.min.js
  three:
  three_waves:
  canvas_lines:
  canvas_sphere:

  # Internal version: 1.0.0
  # canvas_ribbon: //cdn.jsdelivr.net/gh/theme-next/theme-next-canvas-ribbon@1/canvas-ribbon.js
  canvas_ribbon:

# Assets
css: css
js: js
images: images

```

**Hexo and Plugin version(`npm ls --depth 0`)**

```
hexo-site@0.0.0 D:\hexo
+-- hexo@4.2.0
+-- hexo-baidu-url-submit@0.0.6
+-- hexo-blog-encrypt@3.0.12
+-- hexo-deployer-git@2.1.0
+-- hexo-filter-mermaid-diagrams@1.0.5
+-- hexo-generator-archive@1.0.0
+-- hexo-generator-category@1.0.0
+-- hexo-generator-index@1.0.0
+-- hexo-generator-tag@1.0.0
+-- hexo-git-backup@0.1.3
+-- hexo-pdf@1.1.1
+-- hexo-related-popular-posts@4.0.0
+-- hexo-renderer-ejs@1.0.0
+-- hexo-renderer-kramed@0.1.4
+-- hexo-renderer-stylus@1.1.0
+-- hexo-sage-posts@0.0.1
`-- hexo-server@1.0.0

HEXO:
hexo: 4.2.0
hexo-cli: 3.1.0
os: Windows_NT 10.0.18363 win32 x64
node: 12.16.1
v8: 7.8.279.23-node.31
uv: 1.34.0
zlib: 1.2.11
brotli: 1.0.7
ares: 1.15.0
modules: 72
nghttp2: 1.40.0
napi: 5
llhttp: 2.0.4
http_parser: 2.9.3
openssl: 1.1.1d
cldr: 35.1
icu: 64.2
tz: 2019c
unicode: 12.1

```

**Your package.json `package.json`**

```

{
  "name": "hexo-site",
  "version": "0.0.0",
  "private": true,
  "scripts": {
    "build": "hexo generate",
    "clean": "hexo clean",
    "deploy": "hexo deploy",
    "server": "hexo server"
  },
  "hexo": {
    "version": "4.2.0"
  },
  "dependencies": {
    "hexo": "^4.2.0",
    "hexo-baidu-url-submit": "0.0.6",
    "hexo-blog-encrypt": "^3.0.12",
    "hexo-deployer-git": "^2.1.0",
    "hexo-filter-mermaid-diagrams": "^1.0.5",
    "hexo-generator-archive": "^1.0.0",
    "hexo-generator-category": "^1.0.0",
    "hexo-generator-index": "^1.0.0",
    "hexo-generator-tag": "^1.0.0",
    "hexo-git-backup": "^0.1.3",
    "hexo-pdf": "^1.1.1",
    "hexo-related-popular-posts": "^4.0.0",
    "hexo-renderer-ejs": "^1.0.0",
    "hexo-renderer-kramed": "^0.1.4",
    "hexo-renderer-stylus": "^1.1.0",
    "hexo-sage-posts": "0.0.1",
    "hexo-server": "^1.0.0"
  }
}


```

## Others

here's my question.I was using "hexo s" successfullt for a while ,but suddenly it pop out the problem.
my Git is look like :
```
$ hexo s
INFO  Start processing
INFO  Hexo is running at http://localhost:4000 . Press Ctrl+C to stop.
INFO  Congratulations! Your are using the latest version of theme NexT.
INFO  Congratulations! Your are using the latest version of theme NexT.
events.js:288
      throw er; // Unhandled 'error' event
      ^

Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27)
Emitted 'error' event on ClientRequest instance at:
    at TLSSocket.socketErrorListener (_http_client.js:426:9)
    at TLSSocket.emit (events.js:311:20)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  errno: 'ECONNRESET',
  code: 'ECONNRESET',
  syscall: 'read'
}

```
this problem starts after i use "npm update",and it updates "hexo-hide-posts"(this plugin from then on says have problem on "include" so i uninstall it) and "hexo-blog-encrypt"
 these two plugins.this happens while hexo s is running and sometimes it will even appers when using "hexo g -d"
I am really in an embarrassed situation and need help
my repo is https://github.com/dsanying/dsanying.github.io/tree/master and I put some of my files in https://github.com/dsanying/dsanying.github.io/tree/backup

**Comment by seaoak** [comment]
This issue seems duplication of  #4149.

**Comment by dsanying** [comment]
yes that is my question.But how to solve this question?

**Comment by stevenjoezhang** [comment]
Fixed in https://github.com/theme-next/hexo-theme-next/commit/9b543ddacf21a59f2daa52b0ae64075aed62fca2

**Comment by dsanying** [comment]
do I need to clone the theme again or don't need to do anything?

**Comment by dsanying** [comment]
So know I come to another problem.After I change the code ,"git" shows below:
```
ERROR Failed to detect version info. Error message:
ERROR read ECONNRESET
Error: read ECONNRESET
    at TLSWrap.onStreamRead (internal/stream_base_commons.js:205:27)

```
so what leads to this problem happen ?how to fix it ? @stevenjoezhang

**Comment by stevenjoezhang** [comment]
I don't know. Maybe you need a proxy. Can you access this url in the browser?

https://api.github.com/repos/theme-next/hexo-theme-next/releases/latest

**Comment by dsanying** [comment]
not too good in english....Do you mean open?I can see:
```
{
  "url": "https://api.github.com/repos/theme-next/hexo-theme-next/releases/23327289",
  "assets_url": "https://api.github.com/repos/theme-next/hexo-theme-next/releases/23327289/assets",
  "upload_url": "https://uploads.github.com/repos/theme-next/hexo-theme-next/releases/23327289/assets{?name,label}",
  "html_url": "https://github.com/theme-next/hexo-theme-next/releases/tag/v7.7.2",
  "id": 23327289,
  "node_id": "MDc6UmVsZWFzZTIzMzI3Mjg5",
  "tag_name": "v7.7.2",
  "target_commitish": "master",
  "name": "v7.7.2",
  "draft": false,
  "author": {
    "login": "release-drafter[bot]",
    "id": 40829082,
    "node_id": "MDM6Qm90NDA4MjkwODI=",
    "avatar_url": "https://avatars3.githubusercontent.com/in/14356?v=4",
    "gravatar_id": "",
    "url": "https://api.github.com/users/release-drafter%5Bbot%5D",
    "html_url": "https://github.com/apps/release-drafter",
    "followers_url": "https://api.github.com/users/release-drafter%5Bbot%5D/followers",
    "following_url": "https://api.github.com/users/release-drafter%5Bbot%5D/following{/other_user}",
    "gists_url": "https://api.github.com/users/release-drafter%5Bbot%5D/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/release-drafter%5Bbot%5D/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/release-drafter%5Bbot%5D/subscriptions",
    "organizations_url": "https://api.github.com/users/release-drafter%5Bbot%5D/orgs",
    "repos_url": "https://api.github.com/users/release-drafter%5Bbot%5D/repos",
    "events_url": "https://api.github.com/users/release-drafter%5Bbot%5D/events{/privacy}",
    "received_events_url": "https://api.github.com/users/release-drafter%5Bbot%5D/received_events",
    "type": "Bot",
    "site_admin": false
  },
  "prerelease": false,
  "created_at": "2020-03-01T04:10:45Z",
  "published_at": "2020-03-01T04:14:55Z",
  "assets": [

  ],
  "tarball_url": "https://api.github.com/repos/theme-next/hexo-theme-next/tarball/v7.7.2",
  "zipball_url": "https://api.github.com/repos/theme-next/hexo-theme-next/zipball/v7.7.2",
  "body": "## 🌟 New Features\r\n\r\n- Native Dark Mode support (#1386)\r\n\r\n```diff\r\n+ darkmode: true\r\n```\r\n\r\n## ⭐ Features\r\n\r\n- Support MathJax v3.0 (#1314)\r\n- Add next_config helper (#1346)\r\n\r\n## 🐞 Bug Fixes\r\n\r\n- Remove box-shadow of .sidebar when back2top.sidebar is true (#1380)\r\n- Fix config.search.path is null (#1374)\r\n\r\n## 🛠 Improvements\r\n\r\n- Redirect amscd for mathjax3 (#1379)\r\n- Optimize the position of search icon on mobile (#1387)\r\n- Style back-to-top button to make sidebar more comfortable (#1367)\r\n\r\n## 🌀 External Changes\r\n\r\n- [comments] gitalk.swig: fix syntax bug and remove redundant lines introducing access_token (#1369)\r\n\r\n## 📖 Documentation\r\n\r\n- PayPal now supports pay-by-QRCode (#1364)\r\n\r\n***\r\n\r\nFor full changes, see the [comparison between v7.7.1 and v7.7.2](https://github.com/theme-next/hexo-theme-next/compare/v7.7.1...v7.7.2)\r\n"
}
```
Is there anyway to close this js?

**Comment by stevenjoezhang** [comment]
Maybe. This issue has nothing to do with Hexo and theme NexT, please contact your Internet Service Provider or use a proxy.

**Comment by purplezi** [comment]
> not too good in english....Do you mean open?I can see:
> 
> ```
> {
>   "url": "https://api.github.com/repos/theme-next/hexo-theme-next/releases/23327289",
>   "assets_url": "https://api.github.com/repos/theme-next/hexo-theme-next/releases/23327289/assets",
>   "upload_url": "https://uploads.github.com/repos/theme-next/hexo-theme-next/releases/23327289/assets{?name,label}",
>   "html_url": "https://github.com/theme-next/hexo-theme-next/releases/tag/v7.7.2",
>   "id": 23327289,
>   "node_id": "MDc6UmVsZWFzZTIzMzI3Mjg5",
>   "tag_name": "v7.7.2",
>   "target_commitish": "master",
>   "name": "v7.7.2",
>   "draft": false,
>   "author": {
>     "login": "release-drafter[bot]",
>     "id": 40829082,
>     "node_id": "MDM6Qm90NDA4MjkwODI=",
>     "avatar_url": "https://avatars3.githubusercontent.com/in/14356?v=4",
>     "gravatar_id": "",
>     "url": "https://api.github.com/users/release-drafter%5Bbot%5D",
>     "html_url": "https://github.com/apps/release-drafter",
>     "followers_url": "https://api.github.com/users/release-drafter%5Bbot%5D/followers",
>     "following_url": "https://api.github.com/users/release-drafter%5Bbot%5D/following{/other_user}",
>     "gists_url": "https://api.github.com/users/release-drafter%5Bbot%5D/gists{/gist_id}",
>     "starred_url": "https://api.github.com/users/release-drafter%5Bbot%5D/starred{/owner}{/repo}",
>     "subscriptions_url": "https://api.github.com/users/release-drafter%5Bbot%5D/subscriptions",
>     "organizations_url": "https://api.github.com/users/release-drafter%5Bbot%5D/orgs",
>     "repos_url": "https://api.github.com/users/release-drafter%5Bbot%5D/repos",
>     "events_url": "https://api.github.com/users/release-drafter%5Bbot%5D/events{/privacy}",
>     "received_events_url": "https://api.github.com/users/release-drafter%5Bbot%5D/received_events",
>     "type": "Bot",
>     "site_admin": false
>   },
>   "prerelease": false,
>   "created_at": "2020-03-01T04:10:45Z",
>   "published_at": "2020-03-01T04:14:55Z",
>   "assets": [
> 
>   ],
>   "tarball_url": "https://api.github.com/repos/theme-next/hexo-theme-next/tarball/v7.7.2",
>   "zipball_url": "https://api.github.com/repos/theme-next/hexo-theme-next/zipball/v7.7.2",
>   "body": "## 🌟 New Features\r\n\r\n- Native Dark Mode support (#1386)\r\n\r\n```diff\r\n+ darkmode: true\r\n```\r\n\r\n## ⭐ Features\r\n\r\n- Support MathJax v3.0 (#1314)\r\n- Add next_config helper (#1346)\r\n\r\n## 🐞 Bug Fixes\r\n\r\n- Remove box-shadow of .sidebar when back2top.sidebar is true (#1380)\r\n- Fix config.search.path is null (#1374)\r\n\r\n## 🛠 Improvements\r\n\r\n- Redirect amscd for mathjax3 (#1379)\r\n- Optimize the position of search icon on mobile (#1387)\r\n- Style back-to-top button to make sidebar more comfortable (#1367)\r\n\r\n## 🌀 External Changes\r\n\r\n- [comments] gitalk.swig: fix syntax bug and remove redundant lines introducing access_token (#1369)\r\n\r\n## 📖 Documentation\r\n\r\n- PayPal now supports pay-by-QRCode (#1364)\r\n\r\n***\r\n\r\nFor full changes, see the [comparison between v7.7.1 and v7.7.2](https://github.com/theme-next/hexo-theme-next/compare/v7.7.1...v7.7.2)\r\n"
> }
> ```
> 
> Is there anyway to close this js?
> 
> I am in china and I can use proxy omega to open youtube,do you mean this kind of thing? @stevenjoezhang

Do You fix it finally？ 
I bumped into the same problem

**Comment by dsanying** [comment]
Maybe....if you don't want to see this error ,just stop using reminder(reminder: false).In my own opinion,the error appears is because too many requests to the api in a short time.I get some code change so that I can get the reminder of the newest verson while this error don't appear.If you want to know how to change the code get my qq:119548583 for that I don't think I can use English to tell you how to do.

**Comment by purplezi** [comment]
> Maybe....if you don't want to see this error ,just stop using reminder(reminder: false).In my own opinion,the error appears is because too many requests to the api in a short time.I get some code change so that I can get the reminder of the newest verson while this error don't appear.If you want to know how to change the code get my qq:119548583 for that I don't think I can use English to tell you how to do.
> 
> > > not too good in english....Do you mean open?I can see:
> > > ```
> > > {
> > >   "url": "https://api.github.com/repos/theme-next/hexo-theme-next/releases/23327289",
> > >   "assets_url": "https://api.github.com/repos/theme-next/hexo-theme-next/releases/23327289/assets",
> > >   "upload_url": "https://uploads.github.com/repos/theme-next/hexo-theme-next/releases/23327289/assets{?name,label}",
> > >   "html_url": "https://github.com/theme-next/hexo-theme-next/releases/tag/v7.7.2",
> > >   "id": 23327289,
> > >   "node_id": "MDc6UmVsZWFzZTIzMzI3Mjg5",
> > >   "tag_name": "v7.7.2",
> > >   "target_commitish": "master",
> > >   "name": "v7.7.2",
> > >   "draft": false,
> > >   "author": {
> > >     "login": "release-drafter[bot]",
> > >     "id": 40829082,
> > >     "node_id": "MDM6Qm90NDA4MjkwODI=",
> > >     "avatar_url": "https://avatars3.githubusercontent.com/in/14356?v=4",
> > >     "gravatar_id": "",
> > >     "url": "https://api.github.com/users/release-drafter%5Bbot%5D",
> > >     "html_url": "https://github.com/apps/release-drafter",
> > >     "followers_url": "https://api.github.com/users/release-drafter%5Bbot%5D/followers",
> > >     "following_url": "https://api.github.com/users/release-drafter%5Bbot%5D/following{/other_user}",
> > >     "gists_url": "https://api.github.com/users/release-drafter%5Bbot%5D/gists{/gist_id}",
> > >     "starred_url": "https://api.github.com/users/release-drafter%5Bbot%5D/starred{/owner}{/repo}",
> > >     "subscriptions_url": "https://api.github.com/users/release-drafter%5Bbot%5D/subscriptions",
> > >     "organizations_url": "https://api.github.com/users/release-drafter%5Bbot%5D/orgs",
> > >     "repos_url": "https://api.github.com/users/release-drafter%5Bbot%5D/repos",
> > >     "events_url": "https://api.github.com/users/release-drafter%5Bbot%5D/events{/privacy}",
> > >     "received_events_url": "https://api.github.com/users/release-drafter%5Bbot%5D/received_events",
> > >     "type": "Bot",
> > >     "site_admin": false
> > >   },
> > >   "prerelease": false,
> > >   "created_at": "2020-03-01T04:10:45Z",
> > >   "published_at": "2020-03-01T04:14:55Z",
> > >   "assets": [
> > > 
> > >   ],
> > >   "tarball_url": "https://api.github.com/repos/theme-next/hexo-theme-next/tarball/v7.7.2",
> > >   "zipball_url": "https://api.github.com/repos/theme-next/hexo-theme-next/zipball/v7.7.2",
> > >   "body": "## 🌟 New Features\r\n\r\n- Native Dark Mode support (#1386)\r\n\r\n```diff\r\n+ darkmode: true\r\n```\r\n\r\n## ⭐ Features\r\n\r\n- Support MathJax v3.0 (#1314)\r\n- Add next_config helper (#1346)\r\n\r\n## 🐞 Bug Fixes\r\n\r\n- Remove box-shadow of .sidebar when back2top.sidebar is true (#1380)\r\n- Fix config.search.path is null (#1374)\r\n\r\n## 🛠 Improvements\r\n\r\n- Redirect amscd for mathjax3 (#1379)\r\n- Optimize the position of search icon on mobile (#1387)\r\n- Style back-to-top button to make sidebar more comfortable (#1367)\r\n\r\n## 🌀 External Changes\r\n\r\n- [comments] gitalk.swig: fix syntax bug and remove redundant lines introducing access_token (#1369)\r\n\r\n## 📖 Documentation\r\n\r\n- PayPal now supports pay-by-QRCode (#1364)\r\n\r\n***\r\n\r\nFor full changes, see the [comparison between v7.7.1 and v7.7.2](https://github.com/theme-next/hexo-theme-next/compare/v7.7.1...v7.7.2)\r\n"
> > > }
> > > ```
> > > 
> > > 
> > > Is there anyway to close this js?
> > > I am in china and I can use proxy omega to open youtube,do you mean this kind of thing? @stevenjoezhang
> > 
> > 
> > Do You fix it finally？
> > I bumped into the same problem.
> > And I still dont know what to do.
> 
> Maybe....if you don't want to see this error ,just stop using reminder(reminder: false).In my own opinion,the error appears is because too many requests to the api in a short time.I get some code change so that I can get the reminder of the newest verson while this error don't appear.If you want to know how to change the code get my qq:119548583 for that I don't think I can use English to tell you how to do.

okk, thanks a lot
